cmd/kubeadm/app/preflight/checks_unix.go

```diff
@@ -19,15 +19,15 @@ limitations under the License.
 package preflight
 
 import (
-	"fmt"
+	pkgerrors "github.com/pkg/errors"
```

Comment:
again, just change:
```
pkgerrors "github.com/pkg/errors"
```
-> 
```
"github.com/pkg/errors"
```

Reply:
about this,add "github.com/pkg/errors" and then run make test WHAT=./cmd/kubeadm/app...
It shows errors:
cmd/kubeadm/app/preflight/checks_unix.go:22:2: imported and not used: "k8s.io/kubernetes/vendor/github.com/pkg/errors"
cmd/kubeadm/app/preflight/checks_unix.go:30:33: errors.New undefined (type []error has no field or method New)
so add pkgerrors "github.com/pkg/errors" instead of "github.com/pkg/errors".

Reply:
https://github.com/pkg/errors/blob/master/errors.go#L102

this package does have a New function, something about the imports is not correct.
please try to fix it.